src/bitcoin-wallet.cpp

```diff
@@ -94,33 +95,28 @@ int main(int argc, char* argv[])
         return EXIT_FAILURE;
     }
 
-    std::string method {};
-    for(int i = 1; i < argc; ++i) {
-        if (!IsSwitchChar(argv[i][0])) {
-            if (!method.empty()) {
-                tfm::format(std::cerr, "Error: two methods provided (%s and %s). Only one method should be provided.\n", method, argv[i]);
-                return EXIT_FAILURE;
-            }
-            method = argv[i];
-        }
-    }
-
-    if (method.empty()) {
+    std::string method;
+    std::vector<std::string> cmdargs;
+    if (!args.GetCommand(method, cmdargs)) {
```

Comment:
This feels a bit messy; when trying this for bitcoin-util, I added a `struct CmdArg { string cmd; vector<string> args; };` to track them both.

Reply:
Thanks. Fixed, I think